src/app/wishlists/wishlist-file.ts

```diff
@@ -19,6 +19,23 @@ const descriptionLabel = /^@?description:(.+)$/;
  */
 const notesLabel = '//notes:';
 
+const blockNoteLineRegex = /^\/\/notes:(?<blockNotes>[^|]*)/;
+
+/**
+ * Processes wishlist notes to convert escape sequences into actual newlines.
+ * - `\n` becomes a literal newline character
+ * - `\\n` becomes a backslash followed by a newline character
+ * - `<br>` tags are left as-is (for React rendering)
+ */
+function processNotesForDisplay(notes: string | undefined): string | undefined {
+  if (!notes) {
+    return notes;
+  }
+  // Replace \n with actual newline, and \\n with \ followed by newline
```

Comment:
These comments are odd. But in general I guess this is OK, even though it doesn't give users a way to escape out of the escape (e.g. in C-based languages, `\\n` would turn into `\n`)